lib/video_info/providers/youtube.rb

```diff
@@ -63,6 +63,19 @@ def view_count
         _video_statistics['viewCount'].to_i rescue 0
       end
 
+      def author
+        _video_snippet['channelTitle']
+      end
+
+      def author_thumbnail
+        _channel_snippet["items"][0]["snippet"]["thumbnails"]
+        ["default"]["url"] rescue nil
```

Comment:
Avoid using `rescue` in its modifier form.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.